src/app/components/Metadata/__snapshots__/index.test.jsx.snap

```diff
@@ -135,6 +135,9 @@ exports[`Metadata News AMP article should render correctly 1`] = `
     rel="shortcut icon"
     type="image/x-icon"
   />
+  <link
```

Comment:
I would expect this snapshot to have a `href` value, perhaps its not being provided where this snapshot is called?

Reply:
Have updated snapshots.